shared/actions/platform-specific/index.desktop.js

```diff
@@ -361,6 +346,11 @@ export function* platformConfigSaga(): Saga.SagaGenerator<any, any> {
     EngineGen.keybase1NotifySessionClientOutOfDate,
     onOutOfDate
   )
+  getEngine().registerCustomResponse('keybase.1.logsend.prepareLogsend')
```

Comment:
this is how we tell the engine that we intend on responding to these actions ourselves